src/CommonITILTask.php

```diff
@@ -291,10 +291,45 @@ private function handleTaskDuration(array &$input, int $timestart, int $timeend)
         }
     }
 
+    public function assignTechFromtask($input, string $type)
```

Comment:
1. `$type` is not used
2. Please use strict types.
```suggestion
    public function assignTechFromtask(array $input): void
```